CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,93 @@
+Code of Conduct
+===============
+
+The following text is derived from the [Django Project's Code of
+Conduct](https://www.djangoproject.com/conduct/), which is licensed as
+[Creative Commons Attribution](https://creativecommons.org/licenses/by/3.0/).
+
+Like the technical community as a whole, the restic team and community is made
+up of a mixture of professionals and volunteers from all over the world,
+working on every aspect of the mission - including mentorship, teaching, and
+connecting people.
+
+Diversity is one of our huge strengths, but it can also lead to communication
+issues and unhappiness. To that end, we have a few ground rules that we ask
+people to adhere to. This code applies equally to founders, mentors and those
+seeking help and guidance.
+
+This isn't an exhaustive list of things that you can't do. Rather, take it in
+the spirit in which it's intended - a guide to make it easier to enrich all of
+us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the restic project. This
+includes IRC, the issue tracker, the forum, and any other forums created by the
+project team which the community uses for communication. In addition,
+violations of this code outside these spaces may affect a person's ability to
+participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report
+it by emailing conduct AT restic.net.
+
+ * Be friendly and patient.
```

Comment:
I believe there is a section header missing from above this list, that would set the context of what follows. Perhaps "Guiding Principles:" would be a good lead-in so as to suggest these are intended to be read as a well-intended list of principles, but without implying that the list is exhaustive, or that the list should be taken too literally? It really serves as a model for what good behaviour the community should target.